### PR TITLE
add GroupData for Force Touch Events

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

this is part of the #22441 

add *Force Touch Events* to GroupData

fix and update apiref usage

update *See also* section by remove link to *Force Touch events* api summary

### Additional details

see https://developer.apple.com/library/archive/documentation/AppleApplications/Conceptual/SafariJSProgTopics/RespondingtoForceTouchEventsfromJavaScript.html

### CODE REVIEW

File: files/en-us/web/api/element/webkitmouseforcechanged_event/index.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ status:
 browser-compat: api.Element.webkitmouseforcechanged_event
 ---
 
-{{APIRef}}{{Non-standard_header}}
+{{APIRef("Force Touch events")}}{{Non-standard_header}}
 
 The non-standard **`webkitmouseforcechanged`** event is fired by Safari each time the amount of pressure changes on the trackpad/touchscreen.
 
-**`webkitmouseforcechanged`** is a proprietary, WebKit-specific event introduced by Apple to support their [Force Touch events](/en-US/docs/Web/API/Force_Touch_events) feature.
+**`webkitmouseforcechanged`** is a proprietary, WebKit-specific event introduced by Apple to support their {{domxref("Force Touch event")}} feature.
```
Comment (skyclouds2001) [inline files/en-us/web/api/element/webkitmouseforcechanged_event/index.md:15]:
```suggestion
**`webkitmouseforcechanged`** is a proprietary, WebKit-specific event introduced by Apple to support their {{domxref("Force Touch Events")}} feature.
```

File: files/en-us/web/api/element/webkitmouseforcedown_event/index.md [hunk 2]
```diff
@@ -8,11 +8,11 @@ status:
 browser-compat: api.Element.webkitmouseforcedown_event
 ---
 
-{{APIRef}}{{Non-standard_header}}
+{{APIRef("Force Touch events")}}{{Non-standard_header}}
 
 After a {{domxref("Element.mousedown_event", "mousedown")}} event has been fired at the element, if and when sufficient pressure has been applied to the mouse or trackpad button to qualify as a "force click," Safari begins sending **`webkitmouseforcedown`** events to the element.
 
-**`webkitmouseforcedown`** is a proprietary, WebKit-specific event. It is part of the [Force Touch events](/en-US/docs/Web/API/Force_Touch_events) feature.
+**`webkitmouseforcedown`** is a proprietary, WebKit-specific event. It is part of the {{domxref("Force Touch event")}} feature.
```
Comment (skyclouds2001) [inline files/en-us/web/api/element/webkitmouseforcedown_event/index.md:15]:
```suggestion
**`webkitmouseforcedown`** is a proprietary, WebKit-specific event. It is part of the {{domxref("Force Touch Events")}} feature.
```

File: files/en-us/web/api/element/webkitmouseforceup_event/index.md [hunk 3]
```diff
@@ -8,11 +8,11 @@ status:
 browser-compat: api.Element.webkitmouseforceup_event
 ---
 
-{{APIRef}}{{Non-standard_header}}
+{{APIRef("Force Touch events")}}{{Non-standard_header}}
```
Comment (skyclouds2001) [inline files/en-us/web/api/element/webkitmouseforceup_event/index.md:11]:
```suggestion
{{APIRef("Force Touch Events")}}{{Non-standard_header}}
```

File: files/en-us/web/api/element/webkitmouseforcewillbegin_event/index.md [hunk 4]
```diff
@@ -8,15 +8,15 @@ status:
 browser-compat: api.Element.webkitmouseforcewillbegin_event
 ---
 
-{{APIRef}}{{Non-standard_header}}
+{{APIRef("Force Touch events")}}{{Non-standard_header}}
```
Comment (skyclouds2001) [inline files/en-us/web/api/element/webkitmouseforcewillbegin_event/index.md:11]:
```suggestion
{{APIRef("Force Touch Events")}}{{Non-standard_header}}
```

File: files/en-us/web/api/element/webkitmouseforcewillbegin_event/index.md [hunk 5]
```diff
@@ -8,15 +8,15 @@ status:
...
 
 Safari for macOS fires the non-standard **`webkitmouseforcewillbegin`** event at an {{domxref("Element")}} before firing the initial {{domxref("Element/mousedown_event", "mousedown")}} event.
 
-This offers the opportunity to tell the system not to trigger any default Force Touch actions if and when the click turns into a [Force Touch event](/en-US/docs/Web/API/Force_Touch_events).
+This offers the opportunity to tell the system not to trigger any default Force Touch actions if and when the click turns into a {{domxref("Force Touch event")}}.
```
Comment (skyclouds2001) [inline files/en-us/web/api/element/webkitmouseforcewillbegin_event/index.md:15]:
```suggestion
This offers the opportunity to tell the system not to trigger any default Force Touch actions if and when the click turns into a {{domxref("Force Touch Events")}}.
```

File: files/en-us/web/api/element/webkitmouseforcewillbegin_event/index.md [hunk 6]
```diff
@@ -8,15 +8,15 @@ status:
...
 
 To instruct macOS not to engage any default Force Touch actions if the user apply enough pressure to activate a Force Touch event, call {{domxref("Event.preventDefault", "preventDefault()")}} on the `webkitmouseforcewillbegin` event object.
 
-**`webkitmouseforcewillbegin`** is a proprietary, WebKit-specific event. It is part of the [Force Touch events](/en-US/docs/Web/API/Force_Touch_events) feature.
+**`webkitmouseforcewillbegin`** is a proprietary, WebKit-specific event. It is part of the {{domxref("Force Touch event")}} feature.
```
Comment (skyclouds2001) [inline files/en-us/web/api/element/webkitmouseforcewillbegin_event/index.md:19]:
```suggestion
**`webkitmouseforcewillbegin`** is a proprietary, WebKit-specific event. It is part of the {{domxref("Force Touch Events")}} feature.
```

File: files/en-us/web/api/mouseevent/webkit_force_at_force_mouse_down_static/index.md [hunk 7]
```diff
@@ -7,7 +7,7 @@ status:
   - non-standard
 ---
 
-{{APIRef("UI Events")}}{{Non-standard_header}}
+{{APIRef("Force Touch events")}}{{Non-standard_header}}
```
Comment (skyclouds2001) [inline files/en-us/web/api/mouseevent/webkit_force_at_force_mouse_down_static/index.md:10]:
```suggestion
{{APIRef("Force Touch Events")}}{{Non-standard_header}}
```

File: files/en-us/web/api/mouseevent/webkit_force_at_mouse_down_static/index.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ status:
   - non-standard
 ---
 
-{{APIRef("UI Events")}}{{Non-standard_header}}
+{{APIRef("Force Touch events")}}{{Non-standard_header}}
```
Comment (skyclouds2001) [inline files/en-us/web/api/mouseevent/webkit_force_at_mouse_down_static/index.md:10]:
```suggestion
{{APIRef("Force Touch Events")}}{{Non-standard_header}}
```

File: files/en-us/web/api/mouseevent/webkitforce/index.md [hunk 9]
```diff
@@ -7,7 +7,7 @@ status:
   - non-standard
 ---
 
-{{APIRef("UI Events")}}{{Non-standard_header}}
+{{APIRef("Force Touch events")}}{{Non-standard_header}}
```
Comment (skyclouds2001) [inline files/en-us/web/api/mouseevent/webkitforce/index.md:10]:
```suggestion
{{APIRef("Force Touch Events")}}{{Non-standard_header}}
```

**Comment by bsmth** [comment]
All of the changes look good with some things to consider:

- There is no sidebar in https://developer.mozilla.org/en-US/docs/Web/API/Force_Touch_events
- These APIs are vendor-specific (non-standard). I wonder if it's worth having some handling [in Yari](https://github.com/mdn/yari/blob/main/kumascript/macros/APIListAlpha.ejs) that flags these?

**Review by bsmth — APPROVED** [review]
Looks good to me, thank you 👍🏻

**Pull request merged**